integration/src/test/resources/org/mule/test/integration/source-connection-exception-response-handling-test-case.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns:http="http://www.mulesoft.org/schema/mule/http"
+	xmlns="http://www.mulesoft.org/schema/mule/core"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:test="http://www.mulesoft.org/schema/mule/test"
+	xmlns:script="http://www.mulesoft.org/schema/mule/scripting"
+	xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+							 http://www.mulesoft.org/schema/mule/scripting http://www.mulesoft.org/schema/mule/scripting/current/mule-scripting.xsd
+                             http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+                             http://www.mulesoft.org/schema/mule/test http://www.mulesoft.org/schema/mule/test/current/mule-test.xsd">
+	<http:listener-config name="HTTP_Listener_config">
+		<http:listener-connection host="localhost"
+			port="${httpPort}" />
+	</http:listener-config>
+
+	<flow name="http-listener-flow">
+		<http:listener config-ref="HTTP_Listener_config"
+			path="/" />
+		<script:execute engine="groovy">
```

Comment:
why not a `raise`?

Reply:
We need to verify that this is a ConnectionException, so raise is useless. We cannot use test:throw because ConnectionException needs a parameter